web/src/main/java/org/springframework/security/web/DefaultRedirectStrategy.java

```diff
@@ -97,4 +109,12 @@ protected boolean isContextRelative() {
 		return this.contextRelative;
 	}
 
+	public void setStatusCode(HttpStatus statusCode) {
+		this.statusCode = statusCode;
+	}
+
+	protected HttpStatus getStatusCode() {
```

Comment:
Spring Security tends to avoid getters; will you please remove this one for the time being?